app/deck/gateway/configuration.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configuration
```

Comment:
```suggestion
title: Configuring Authentication
```
"Configuration" is much too general a title and tells me nothing about this page.